### ISSUE TITLE
Create a contact page (conventional routing)

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#39]
### Who

- [x] Anonymous visitor
- [ ] Patient
- [ ] Doctor
- [ ] Content editor
- [ ] Marketer
- [ ] Administrator
- [ ] Developer

### Needs what

A page with contact information, together with a simple image gallery.

### Why

Why is the feature needed?

### Issue expands upon

This issue expands upon another issue(s): #`issue number`

### Blockers

This issue is blocked by the following issue(s): #9
